File: my_blog_前端/lib/model/Comment.dart
import 'package:flutter/material.dart';

class Comment {
  int? id; // 评论序号
  String? author; // 文章作者名
  String? title; // 文章标题
  String? label; // 文章标签
  String? username; // 评论者
  String? content; // 评论内容
  String? releaseTime; // 评论时间

  Comment({
    this.id,
    this.author,
    this.title,
    this.label,
    this.username,
    this.content,
    this.releaseTime,
  });

  Comment.fromJson(dynamic json) {
    id = json["id"];
    author = json["author"];
    title = json["title"];
    label = json["label"];
    username = json["username"];
    content = json["content"];
    releaseTime = json["releaseTime"];
  }

  Map<String, dynamic> toJson() {
    var map = <String, dynamic>{};
    map["id"] = id;
    map["author"] = author;
    map["title"] = title;
    map["label"] = label;
    map["username"] = username;
    map["content"] = content;
    map["releaseTime"] = releaseTime;
    return map;
  }
}

File: myBlog_后端/src/main/java/com/tianyufighter/myblog/model/Comment.java
package com.tianyufighter.myblog.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Comment {
    private int id;
    private String author; // 文章作者名
    private String title; // 文章标题
    private String label; // 文章标签
    private String username; // 评论者
    private String content; // 评论内容
    private Date releaseTime; // 评论时间
    public Comment(String author, String title, String label, String username, String content, Date releaseTime) {
        this.author = author;
        this.title = title;
        this.label = label;
        this.username = username;
        this.content = content;
        this.releaseTime = releaseTime;
    }
}
